src/server/database/migrations/20171115161524-fixRemovedTeamMember.js

```diff
@@ -0,0 +1,10 @@
+exports.up = async (r) => {
+  await r.table('Project')
+    .update((project) => ({
```

Comment:
Do we want to filter to the subset of project cards which need to be updated? Or is this safe to do regardless?

Reply:
for updates, rethinkdb always tells you if the record was updated or not. That means they either compare the updates to the current record before or after the update. I'm assuming they check it before (to avoid a write). if that's the case, then this would actually be slightly more performant than iterating over every record for a filter & then iterating over that subset. i never benchmarked it though, i could be wrong!

Reply:
That's cool. I'm not too concerned about performance for a migration.